handler.go

```diff
@@ -210,19 +227,21 @@ func CreateComment(w http.ResponseWriter, req *http.Request) {
 }
 
 type userClaims struct {
-	ID bson.ObjectId
+	ID    bson.ObjectId
```

Comment:
you could embed the user struct here: https://play.golang.org/p/2jZMCGEfxW

Reply:
reverting last commit that removed the embedded struct